README.rst

```diff
@@ -30,6 +30,17 @@ Or from the command line::
 
 Enjoy!
 
+Addition/Removal of Certificates
+--------------------------------
+
+Certify does not support any addition/removal or other modification of the
+CA trust store content. This project is intended to provide a reliable and
+highly portable root of trust to python deployments.
+
+Look to upstream projects using Certify for methods to use alternate trust
```

Comment:
Given this is documented in those projects, this seems unnecessary as those aren't the only projects using this library which you've also misspelled several times.

Reply:
Fixed spelling. Sorry.

This will help others, and Requests is already mentioned in this README, hence including it here.